packages/now-node/src/launcher.ts

```diff
@@ -8,108 +12,148 @@ type LauncherConfiguration = {
     }
-    bridge.setServer(server);
-    bridge.listen();
-  } else if (typeof listener === 'object' && Object.keys(listener).length === 0) {
-    setTimeout(() => {
-      if (!isServerListening) {
-        console.error('No exports found in module "${entrypointPath}".');
-        console.error('Did you forget to export a function or a server?');
-        process.exit(1);
+
+    try {
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      let listener = require(entrypointPath);
+      if (listener.default) listener = listener.default;
+
+      if (typeof listener.listen === 'function') {
+        Server.prototype.listen = originalListen;
+        const server = listener;
+        bridge.setServer(server);
+        bridge.listen();
+      } else if (typeof listener === 'function') {
+        Server.prototype.listen = originalListen;
+        let server: Server;
+        if (shouldAddHelpers) {
+          bridge = new Bridge(undefined, true);
+          server = require(helpersPath).createServerWithHelpers(
+            listener,
+            bridge
+          );
+        } else {
+          server = createServer(listener);
+        }
+        bridge.setServer(server);
+        bridge.listen();
+      } else if (
+        typeof listener === 'object' &&
+        Object.keys(listener).length === 0
+      ) {
+        setTimeout(() => {
+          if (!isServerListening) {
+            console.error('No exports found in module %j.', entrypointPath);
+            console.error('Did you forget to export a function or a server?');
+            process.exit(1);
+          }
+        }, 5000);
+      } else {
+        console.error('Invalid export found in module %j.', entrypointPath);
+        console.error('The default export must be a function or server.');
       }
-    }, 5000);
-  } else {
-    console.error('Invalid export found in module "${entrypointPath}".');
-    console.error('The default export must be a function or server.');
-  }
-} catch (err) {
-  if (err.code === 'MODULE_NOT_FOUND') {
-    console.error(err.message);
-    console.error('Did you forget to add it to "dependencies" in \`package.json\`?');
-    process.exit(1);
-  } else {
-    console.error(err);
-    process.exit(1);
-  }
+    } catch (err) {
+      if (err.code === 'MODULE_NOT_FOUND') {
+        console.error(err.message);
+        console.error(
+          'Did you forget to add it to "dependencies" in `package.json`?'
+        );
+      } else {
+        console.error(err);
+      }
+      process.exit(1);
+    }
+
+    return bridge;
+  };
 }
 
-exports.launcher = bridge.launcher;`;
+export function makeAwsLauncher(config: LauncherConfiguration): string {
+  const { entrypointPath, awsLambdaHandler = '' } = config;
+  return `const url_1 = require("url");
+const funcName = ${JSON.stringify(awsLambdaHandler.split('.').pop())};
+const entrypointPath = ${JSON.stringify(entrypointPath)};
+exports.launcher = ${getAwsLauncher(config)}`;
```

Comment:
Does this work? I would think you would have to do `getAwsLauncher.toString()` to stringify the function.

Reply:
`${}` calls `String()` implicitly.